### PR TITLE
Master to QC

### PR DESCRIPTION
**Pull request by mklauber**

### CODE REVIEW

File: seo/tests/test_views.py [hunk 1]
```diff
@@ -2865,6 +2865,72 @@ def test_invalid_moc_404(self):
                                HTTP_HOST=self.site.domain)
         self.assertEqual(resp.status_code, 404)
 
+class FilterTestCase404(DirectSEOTestCase):
```
Comment (tdphillips) [inline seo/tests/test_views.py]: This is the only serious conflict I see. There are two classes with this name here and in QC. Both of these subclass DirectSEOTestCase while one of the ones in QC subclasses a different test case.

File: registration/views.py [hunk 2]
```diff
@@ -150,7 +150,6 @@ def merge_accounts(request, activation_key):
         status.approved_by = existing_user
         status.save()
 
-
```
Comment (mklauber) [inline registration/views.py]: This one was me.  I made a pep8 change on the staging branch to trigger a new build.

**Comment by aspidites** [comment]
I'm a little concerned with the fact that changes which aren't changes are being picked up, in addition to the bidirectional merging we seem to be doing (QC -> Staging -> Master -> QC).

**Comment by mklauber** [comment]
Typically, we aren't doing master->QC.  I dropped this in right now, just to see what changes we had, and was rather surprised at them.

File: mypartners/views.py [hunk 3]
```diff
@@ -91,7 +91,7 @@ def prm(request):
         'company': company,
         'user': request.user,
         'partner_ct': ContentType.objects.get_for_model(Partner).id,
-        'view_name': 'PRM',
+        'view_name': 'PRM'
```
Comment (mklauber) [inline mypartners/views.py]: Looks like this was a change that David made, that was never merged back to QC.  https://github.com/DirectEmployers/MyJobs/commit/0507be7aa70345f8fd83fa4af032edb4479d0fbf

File: myreports/views.py [hunk 4]
```diff
@@ -135,7 +135,7 @@ class ReportView(View):
     app = 'mypartners'
     model = 'contactrecord'
 
-    @requires(PRM, missing_activity, missing_access)
+    @method_decorator(requires(PRM, missing_activity, missing_access))
```
Comment (mklauber) [inline myreports/views.py]: I think the cause of this (despite the fact that the code is in fact present on QC) is that the code change was merged to staging from a different branch (https://github.com/DirectEmployers/MyJobs/pull/1852) than the code merged to quality control (https://github.com/DirectEmployers/MyJobs/pull/1851).  Not sure why this confused git other than the commit tree cannot be resolved simply?

File: seo/tests/test_views.py (hunk 1 above)
Comment (mklauber) [inline seo/tests/test_views.py]: The original cause of this looks like it was my fault.  I think it was probably a merge conflict that was improperly resolved.

File: wsgi.py [hunk 5]
```diff
@@ -4,7 +4,7 @@
 newrelic.agent.initialize('/home/web/MyJobs/MyJobs-urls/newrelic.ini')
 
 
-os.environ['DJANGO_SETTINGS_MODULE'] = 'MyJobs-urls.settings'
+os.environ['DJANGO_SETTINGS_MODULE'] = 'settings'
```
Comment (mklauber) [inline wsgi.py]: I think the cause of this (despite the fact that the code is in fact present on QC) is that the code change was merged to staging from a different branch (#1847) than the code merged to quality control (#1845). Not sure why this confused git other than the commit tree cannot be resolved simply?

File: myreports/views.py (hunk 4 above)
Comment (aspidites) [inline myreports/views.py]: Hmm...I don't think this has ever caused issues before. I'd be a bit sad if git(hub) wasn't able to see that two files, regardless of which commit they were changed in, were similar.

File: wsgi.py (hunk 5 above)
Comment (aspidites) [inline wsgi.py]: Same comment as above.

**Comment by aspidites** [comment]
I'm fine with this being merged even though it makes me feel dirty. I wonder if some experimentation on a test repo might uncover exactly what pattern of behavior causes these results? Should be easier to inspect than one with a ton of history like this. 

I don't think doing so should be a priority, however, as you've got way more important things on your plate, but it could be an interesting experiment none-the-less.

**Comment by mklauber** [comment]
test this please

**Pull request merged**